packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/simulationservice/SimulationResource.java

```diff
@@ -0,0 +1,56 @@
+package software.uncharted.terarium.hmiserver.resources.simulationservice;
+
+import io.quarkus.security.Authenticated;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import software.uncharted.terarium.hmiserver.models.simulationservice.SimulationParams;
+import software.uncharted.terarium.hmiserver.proxies.simulationservice.SimulationServiceProxy;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/api/simulation-service")
+@Authenticated
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "Simulation Service REST Endpoint")
+public class SimulationResource {
+
+	@RestClient
+	SimulationServiceProxy proxy;
+
+	@POST
+	@Path("/forecast")
+	@Produces(MediaType.APPLICATION_JSON)
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Tag(name = "Make new forecast simulation")
+	public Response makeForecastRun(
+		final SimulationParams simulationParams
+	) {
+		System.out.println(simulationParams);
+		System.out.println(proxy);
```

Comment:
I think we should remove logs, unless, if you want to see this information it will be better to log them properly to see them in Grafana.

Reply:
whoops forgot to remove these, was debugging